schedule-cifar100.yaml

```diff
@@ -1,12 +1,12 @@
 lr_schedulers:
```

Comment:
This file looks similar to "schedule-cifar-nas.yaml". We may need to remove one of them in the next commit.